### ISSUE TITLE
Using `{}` syntax for OR conditions in inputs does not work

### ISSUE THREAD

**Issue by sppatel** [vercel/turborepo#1249]
### What version of Turborepo are you using?

1.2.9

### What package manager are you using / does the bug impact?

Yarn v1

### What operating system are you using?

Mac

### Describe the Bug

When supply inputs to consider specific file extensions.  The following works.

```
{
   "inputs":[
      "**/*.js",
      "**/*.ts"
   ]
}
```

However when the consolidating these into a single entry using `{}` glob syntax, any changes matching those extensions does not bust the cache.

```
{
   "inputs":[
      "**/*.{js, ts}"
   ]
}
```

### Expected Behavior

Using `{}` syntax is fairly common and would expect this to work.

### To Reproduce

Scope inputs to specific individual file extensions as described in the bug description.  Verify that the behavior works as expected.  Now conslidate the entries in inputs to a single entry using an OR condition syntax. e.g `"**/*.{js, ts}"`.  Make modifications to any ts or js file in that package and you'll see that the fingerprint does not change.

**Comment by dobesv** [comment]
inputs use a different globbing syntax than outputs.  I've opened a PR to document this here: 

https://github.com/vercel/turborepo/pull/1690

**Comment by gsoltis** [comment]
This is fixed in #1699, which has been released with version 1.4.4
